bots/jabber/network.js

```diff
@@ -0,0 +1,2 @@
+const xmpp = Symbol.for('XMPP');
+module.exports = xmpp;
```

Comment:
Just an idea: let's place all the networks in a single file? I can't see a reason to keep a bunch of files 2 lines long.

Reply:
The idea was that outside world doesn't need to know anything about the kind of network, this is needed just to skip double handling of messages sent through bus.

Reply:
Eh, okay then.